### DISCUSSION TITLE
Any idea what's causing New Branch dialog to render this way (master)

### DISCUSSION THREAD

**Discussion by mdonatas** [gitextensions/gitextensions#9404] [General]
I had a quick look at the code but couldn't fine the culprit. Has anyone else seen this with other forms or have hints as to what might be happening?

`master` branch
![image](https://user-images.githubusercontent.com/483659/126604284-d385cf99-45ed-4b58-98b5-16c3bbe1801b.png)

**Comment by mdonatas** [comment]
p.s. is this a good place for such questions or would you say Gitter is more suitable?

**Reply by pmiossec** [reply]
I reproduce. Please provide your dpi setting.

I have reported the same kind of broken ui for push form yesterday: #9402

**Reply by mstv** [reply]
> is this a good place for such questions or would you say Gitter is more suitable?

It is an issue of type bug. Let's keep it in GH.
